readthedocs/embed/views.py

```diff
@@ -309,9 +309,15 @@ def parse_sphinx(content, section, url):
     for element_id in elements_id:
         if not element_id:
             continue
-        query_result = body_obj(f'#{element_id}')
-        if query_result:
-            break
+        try:
+            query_result = body_obj(f'#{element_id}')
+            if query_result:
+                break
+        except Exception:  # noqa
```

Comment:
Can we make the exception we catch here more specific?

Reply:
the exception comes from a third party package that is used from pyquery, I don't like having imports from packages that aren't listed in our requirements, also there may be more exceptions than the one reported, so we would still need to have a broader check. And, only the pyquery execution is inside the try block, so no need to scope it down.